include/VulkanInfoProvider.hpp

```diff
@@ -3,19 +3,20 @@
 #include <lug/Config.hpp>
 #include <lug/Graphics/Renderer.hpp>
 #include <lug/Graphics/Vulkan/Renderer.hpp>
+#include <lug/Graphics/Vulkan/API/RTTI/Enum.hpp>
 
 #if defined(LUG_SYSTEM_WINDOWS)
-    #pragma warning(push)
-    #pragma warning(disable : 4003)
+#pragma warning(push)
+#pragma warning(disable : 4003)
 #endif
 #include <json/json.hpp>
 #if defined(LUG_SYSTEM_WINDOWS)
-    #pragma warning(pop)
+#pragma warning(pop)
 #endif
 
 class VulkanInfoProvider {
```

Comment:
For the API, it's a GPU entry and not just vulkan information (and later we will have an entry for the computer with some information like the RAM, the CPU, etc and each computer will have one or more GPUs). So, can we rename this class `GPUInfoProvider` for example?